### PR TITLE
hotspot: init at 1.0.0

### PR DESCRIPTION
**Pull request by nh2**
###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by 0xABAB — APPROVED** [review]

File: pkgs/development/tools/analysis/hotspot/default.nix [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    ki18n
+    kitemmodels
+    kitemviews
+    libelf
+    qtbase
+    threadweaver
+  ];
+  nativeBuildInputs = [ makeQtWrapper ];
+
+  patchPhase = ''
```
Comment (FRidh) [inline pkgs/development/tools/analysis/hotspot/default.nix]: `postPatch`. Try not to override default phases.

**Comment by nh2** [comment]
I need to revise this due to commit f3ce85235 which removed `makeQtWrapper` (apparently it is no longer needed) (I cannot find _why_ it is no longer needed though, maybe you can enlighten me @ttuegel).

File: pkgs/development/tools/analysis/hotspot/default.nix (hunk 1 above)
Comment (nh2) [inline pkgs/development/tools/analysis/hotspot/default.nix]: Fixed.

**Comment by nh2** [comment]
> I need to revise this due to commit f3ce852 which removed makeQtWrapper

Done. Please take another look.

**Comment by bjornfor** [comment]
Builds and runs. Applied to master (a37f3a3573bfd39b00ece2ec3f00661b020ad14c, 35e912793ac5b4c6e808dc23e61d755d473580ed). Thanks!

**Pull request closed without merging**